ivy/functional/frontends/jax/numpy/creation.py

```diff
@@ -15,14 +15,17 @@
 @handle_jax_dtype
 @to_ivy_arrays_and_back
 def array(object, dtype=None, copy=True, order="K", ndmin=0):
-    # TODO must ensure the array is created on default device.
     if order is not None and order != "K":
         raise ivy.utils.exceptions.IvyNotImplementedException(
             "Only implemented for order='K'"
         )
     ret = ivy.array(object, dtype=dtype)
     if ivy.get_num_dims(ret) < ndmin:
         ret = ivy.expand_dims(ret, axis=list(range(ndmin - ivy.get_num_dims(ret))))
+
+    default_device = ivy.get_default_dtype
```

Comment:
Hi @sladyn98, have you tried running the function's test? It is currently failing due to using the wrong function. I think you should be using `ivy.get_default_device()`.